Jenkinsfile

```
@@ -465,12 +475,31 @@ def e2e(Map args = [:]) {
   }
 }
 
+/**
+* This method is a wrapper to run the runCommand method and retry if there are 
+* environmental issues. Therefore it passes the arguments to the runCommand.
+* For further details regarding the arguments please refers to the runCommand method.
+*/
+def target(Map args = [:]) {
```

What do think about a more descriptive name? I acknowledge references should be updated.
```suggestion
def safeRunCommand(Map args = [:]) {
```
Alternatives?
- runCommandWithEnviromentalIssues _(longer)_

---

I was not intended to change the name at first, though it might be worth to add more clarity.

What do you think `runCommandWithRetry`?